tensorboard/components/tf_trace_viewer/tf-trace-viewer.html

```diff
@@ -15,6 +15,75 @@
 limitations under the License.
 -->
 
+<!-- Before any Trace Viewer scripts run, show warning if needed. -->
```

Comment:
Please note that this is meant to work without Polymer and hence cannot be part of Polymer component.

Reply:
Sounds good to call out.  I've updated the comment.